public/app/features/dashboard-scene/utils/urlBuilders.ts

```diff
@@ -62,6 +62,8 @@ export function getDashboardUrl(options: DashboardUrlOptions) {
 
   const params = options.currentQueryParams ? locationSearchToObject(options.currentQueryParams) : {};
 
+  delete params['shareView'];
```

Comment:
Can we add also a simple test in `urlBuilders.test.ts`?
Are we sure this `getDashboardurl` function will never need to return a `shareView` param? I think the answer is yes but I want to double check just in case.

Reply:
Test added!

Seems like shareView is used to open the different tabs in the share modal. When copying link to share, we don't need that info